source/Meadow-Filter-Parse.js

```diff
@@ -160,6 +160,47 @@ const addFilterStanzaToQuery = (pFilterStanza, pQuery) =>
 			}
 			break;
 
+    case 'FBJV':// Filter by JSON Value (left-side AND)
+      {
+      const fields = pFilterStanza.Field.split(',', 2);
+      const jsonField =fields[0];
+      const subFields = fields[1];
+      if (jsonField !== undefined && jsonField.trim() !== "" && subFields !== undefined && subFields.trim() !== "")
+        {
+      pQuery
+        .addFilter('', '1', `LENGTH(${jsonField}) >`, 'AND', 'jsonFieldLen')
+        .addFilter(
```

Comment:
Since we are adding two clauses here in one, I think we need to add parentheses, otherwise the input query may be broken.